### ISSUE TITLE
Unnecessary Clang `macro is not used [Lexical or Preprocessor Issue]` warning

### ISSUE THREAD

**Issue by lukaszpolowczyk** [jbenden/vscode-c-cpp-flylint#23]
I have a simple first two lines in *.h file:
```
#ifndef serialize_orz_H
#define serialize_orz_H
```
Clang return warning that: `macro is not used [Lexical or Preprocessor Issue]`
Actually it is use, in first line.
This is bug? How use/config Clang or how write this two lines, to not see this warning?

**Comment by lukaszpolowczyk** [comment]
I used option: `"c-cpp-flylint.clang.extraArgs": ["Wunused-macros"]`.
But still I think this is bad solution.
This flag blocking all warnings about not used macro.
I want blocked only this one type of case.
How do that?

**Comment by jbenden** [comment]
Try the following setting for your workspace:

```
    "c-cpp-flylint.clang.warnings": [
        "all",
        "extra"
    ]
```
It will reduce the amount of reported problems.

Best Regards,
-Joe
